workflows/genomeassembly.nf

```diff
@@ -6,9 +6,6 @@
 
 def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
 
-// Validate input parameters
-WorkflowGenomeassembly.initialise(params, log)
```

Comment:
Is that really needed ? I wonder if this line was failing because the file has been renamed